### PR TITLE
Clarify GiB unit for storage

### PR DESCRIPTION
**Pull request by michrome**

### CODE REVIEW

**Review by danieltprice — APPROVED** [review]

File: content/docs/import/import-from-neon.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ This guide describes how to migrate a database from one Neon project to another
...
-- **Piping considerations**: Piping is not recommended for medium (> 1 GB) and large databases (> 5 GB), as it is susceptible to failures during lengthy migration operations (see [Pipe pg_dump to pg_restore](/docs/import/import-from-postgres#pipe-pgdump-to-pgrestore) for more information). If your database is larger than 1 GB, we recommend performing the dump and restore as separate operations. For instructions, see [Import data from Postgres](/docs/import/import-from-postgres).
+- **Piping considerations**: Piping is not recommended for medium (> 1 GiB) and large databases (> 5 GiB), as it is susceptible to failures during lengthy migration operations (see [Pipe pg_dump to pg_restore](/docs/import/import-from-postgres#pipe-pgdump-to-pgrestore) for more information). If your database is larger than 1 GiB, we recommend performing the dump and restore as separate operations. For instructions, see [Import data from Postgres](/docs/import/import-from-postgres).
```
Comment (stepashka) [inline content/docs/import/import-from-neon.md:15]: FWIW i don't think it's critical to have GiB here.
up to docs team to decide

File: content/docs/import/import-from-postgres.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ Repeat the `pg_dump` and `pg_restore` process for each database you want to migr
    </CodeBlock>
 
 - Consider running a test migration first to ensure your actual migration goes smoothly. See [Run a test migration](#run-a-test-migration).
-- If your database is small (< 1 GB), you can pipe `pg_dump` output directly to `pg_restore` to save time. See [Pipe pg_dump to pg_restore](#pipe-pg_dump-to-pg_restore).
+- If your database is small (< 1 GiB), you can pipe `pg_dump` output directly to `pg_restore` to save time. See [Pipe pg_dump to pg_restore](#pipe-pg_dump-to-pg_restore).
```
Comment (stepashka) [inline content/docs/import/import-from-postgres.md:32]: FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

File: content/docs/import/import-from-postgres.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ mydumpfile.bak
 
 ## Pipe pg_dump to pg_restore
 
-For small databases (< 1 GB), the standard output of `pg_dump` can be piped directly into a `pg_restore` command to minimize migration downtime:
+For small databases (< 1 GiB), the standard output of `pg_dump` can be piped directly into a `pg_restore` command to minimize migration downtime:
```
Comment (stepashka) [inline content/docs/import/import-from-postgres.md:99]: FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

File: content/docs/import/import-from-postgres.md [hunk 4]
```diff
@@ -112,7 +112,7 @@ pg_dump -Fc -v -d <source_database_connection_string> | pg_restore -v -d <neon-d
 
 </CodeBlock>
 
-Piping is not recommended for medium (> 1 GB) and large databases (> 5 GB), as it is susceptible to failures during lengthy migration operations.
+Piping is not recommended for medium (> 1 GB) and large databases (> 5 GiB), as it is susceptible to failures during lengthy migration operations.
```
Comment (stepashka) [inline content/docs/import/import-from-postgres.md:115]: FWIW i don't think it's critical to have GiB here.

up to docs team to decide

File: content/docs/import/import-from-postgres.md [hunk 5]
```diff
@@ -149,13 +149,13 @@ The `pg_dump` and `pg_restore` commands provide numerous advanced options, some
...
 - `--lock-wait-timeout=20s`: Error out early in the dump process instead of waiting for an unknown amount of time if there is lock contention.
 Do not wait forever to acquire shared table locks at the beginning of the dump. Instead fail if unable to lock a table within the specified timeout.`
-- `-j <njobs>`: Consider this option for medium (>1 GB) and large databases (>5) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
+- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
```
Comment (stepashka) [inline content/docs/import/import-from-postgres.md:152]:
```suggestion
- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5 GiB) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
```

Comment (stepashka) [reply content/docs/import/import-from-postgres.md:152]: btw, i did not realize we suggest this classification on medium and large!
do you remember, @danieltprice , where did we get these definitions from?

File: content/docs/import/import-from-postgres.md [hunk 6]
```diff
@@ -149,13 +149,13 @@ The `pg_dump` and `pg_restore` commands provide numerous advanced options, some
...
 
 ### pg_restore options
 
 - `-c --if-exists`: Drop database objects before creating them if they already exist. If you had a failed migration, you can use these options to drop objects created by the previous migration to avoid errors when retrying the migration.
-- `-j <njobs>`: Consider this option for medium (>1 GB) and large databases (>5) to run the restore process in parallel. Set `<njobs>` to the number of available vCPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only makes sense for Pro plan users who can configure computes with >1 vCPU. It cannot be used together with `--single-transaction`.
+- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5) to run the restore process in parallel. Set `<njobs>` to the number of available vCPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only makes sense for Pro plan users who can configure computes with >1 vCPU. It cannot be used together with `--single-transaction`.
```
Comment (stepashka) [inline content/docs/import/import-from-postgres.md:158]:
```suggestion
- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5 GiB) to run the restore process in parallel. Set `<njobs>` to the number of available vCPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only makes sense for Pro plan users who can configure computes with >1 vCPU. It cannot be used together with `--single-transaction`.
```

File: content/docs/import/import-from-postgres.md (hunk 5 above)
Comment (stepashka) [reply content/docs/import/import-from-postgres.md:152]: FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

File: content/docs/import/import-from-postgres.md (hunk 6 above)
Comment (stepashka) [reply content/docs/import/import-from-postgres.md:158]: FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

File: content/docs/import/import-sample-data.md [hunk 7]
```diff
@@ -13,12 +13,12 @@ This guide describes how to download and install sample data for use with Neon.
 - A `psql` client for connecting to your Neon database and loading data. This client is included with a standalone PostgreSQL installation. See [PostgreSQL Downloads](https://www.postgresql.org/download/).
 - A `pg_restore` client if you are loading the [employees](#employees-database) or [postgres_air](#postgres-air-database) database. The `pg_restore` client is included with a standalone PostgreSQL installation. See [PostgreSQL Downloads](https://www.postgresql.org/download/).
 - A Neon database connection string. After creating a database, you can obtain the connection string from the **Connection Details** widget on the Neon **Dashboard**. In the instructions that follow, replace `postgres://[user]:[password]@[neon_hostname]/[dbname]` with your connection string.
-- A Neon [Pro](/docs/introduction/pro-plan) account if you intend to install a dataset larger than 3 GBs.
+- A Neon [Pro](/docs/introduction/pro-plan) account if you intend to install a dataset larger than 3 GiBs.
```
Comment (stepashka) [inline content/docs/import/import-sample-data.md:16]: concerned about the S in the end of GiB**s**!

File: content/docs/import/import-sample-data.md [hunk 8]
```diff
@@ -13,12 +13,12 @@ This guide describes how to download and install sample data for use with Neon.
...
 - Instructions for each dataset require that you create a database. You can do so from a client such as `psql` or from the [Neon SQL Editor](/docs/get-started-with-neon/query-with-neon-sql-editor).
 
 ## Sample data
 
-Sample datasets are listed in order of the smallest to largest installed size. Please be aware that the Neon Free Tier has a storage limit of 3 GBs per branch. Datasets larger than 3 GBs cannot be loaded on the Free Tier.
+Sample datasets are listed in order of the smallest to largest installed size. Please be aware that the Neon Free Tier has a storage limit of 3 GiBs per branch. Datasets larger than 3 GiBs cannot be loaded on the Free Tier.
```
Comment (stepashka) [inline content/docs/import/import-sample-data.md:21]: concerned about the S in the end of GiBs!

File: content/release-notes/2022-12-06-console.md [hunk 9]
```diff
@@ -17,5 +17,5 @@
 - UI: Updated the passwordless auth feature to show the branch name on the browser page that is displayed during authentication.
 - UI: Updated the **Create branch** page to display the local timezone when selecting the **Time** option during branch creation.
 - UI: Removed branches without endpoints from the branch selector in Neon SQL Editor. To query a branch, the branch must have an associated endpoint through which a connection can be established.
-- UI: Added a **Free tier** widget to the Neon **Dashboard**, which shows how many branches and endpoints you have created and the status of your free tier limits. The widget also shows the data size limit per branch, which is 3GB. For information about Free Tier limits, see [Neon Technical Preview Free Tier](/docs/introduction/free-tier/).
+- UI: Added a **Free tier** widget to the Neon **Dashboard**, which shows how many branches and endpoints you have created and the status of your free tier limits. The widget also shows the data size limit per branch, which is 3GiB. For information about Free Tier limits, see [Neon Technical Preview Free Tier](/docs/introduction/free-tier/).
```
Comment (stepashka) [inline content/release-notes/2022-12-06-console.md:20]:
```suggestion
- UI: Added a **Free tier** widget to the Neon **Dashboard**, which shows how many branches and endpoints you have created and the status of your free tier limits. The widget also shows the data size limit per branch, which is 3 GiB. For information about Free Tier limits, see [Neon Technical Preview Free Tier](/docs/introduction/free-tier/).
```

File: content/release-notes/2022-12-28-console.md [hunk 10]
```diff
@@ -1,6 +1,6 @@
 ### Fixes & improvements
 
-- API: Added a `branch_logical_size_limit` attribute to the Neon API [Project](https://api-docs.neon.tech/reference/getproject) response body, which shows the logical data size limit for project branches. This attribute replaces the `logical_size_limit` attribute, which was shown in the [Branch](https://api-docs.neon.tech/reference/getprojectbranch) response body. The [Neon Free Tier](/docs/introduction/free-tier/) limits the logical data size of a branch to 3072MB (3GB).
+- API: Added a `branch_logical_size_limit` attribute to the Neon API [Project](https://api-docs.neon.tech/reference/getproject) response body, which shows the logical data size limit for project branches. This attribute replaces the `logical_size_limit` attribute, which was shown in the [Branch](https://api-docs.neon.tech/reference/getprojectbranch) response body. The [Neon Free Tier](/docs/introduction/free-tier/) limits the logical data size of a branch to 3072MB (3GiB).
```
Comment (stepashka) [inline content/release-notes/2022-12-28-console.md:3]:
```suggestion
- API: Added a `branch_logical_size_limit` attribute to the Neon API [Project](https://api-docs.neon.tech/reference/getproject) response body, which shows the logical data size limit for project branches. This attribute replaces the `logical_size_limit` attribute, which was shown in the [Branch](https://api-docs.neon.tech/reference/getprojectbranch) response body. The [Neon Free Tier](/docs/introduction/free-tier/) limits the logical data size of a branch to 3072 MB (3 GiB).
```

File: src/components/pages/ai/testimonials/testimonials.jsx [hunk 11]
```diff
@@ -13,7 +13,7 @@ import TwitterIcon from './images/twitter.inline.svg';
...
+    text: 'What if PGVector was on steroids ?! <a href="https://twitter.com/raoufdevrel" target="_blank" rel="noreferrer noopener">@raoufdevrel</a> from <a href="https://twitter.com/neondatabase" target="_blank" rel="noreferrer noopener">@neondatabase</a> just did that with pg_embedding and made it available through <a href="https://twitter.com/LangChainAI" target="_blank" rel="noreferrer noopener">@LangChainAI</a>. I&apos;ll definitely give it a try on <a href="https://twitter.com/quivr_brain" target="_blank" rel="noreferrer noopener">@quivr_brain</a> and our 10GiB of vectors that use PGVector',
```
Comment (stepashka) [inline src/components/pages/ai/testimonials/testimonials.jsx:16]:
```suggestion
    text: 'What if PGVector was on steroids ?! <a href="https://twitter.com/raoufdevrel" target="_blank" rel="noreferrer noopener">@raoufdevrel</a> from <a href="https://twitter.com/neondatabase" target="_blank" rel="noreferrer noopener">@neondatabase</a> just did that with pg_embedding and made it available through <a href="https://twitter.com/LangChainAI" target="_blank" rel="noreferrer noopener">@LangChainAI</a>. I&apos;ll definitely give it a try on <a href="https://twitter.com/quivr_brain" target="_blank" rel="noreferrer noopener">@quivr_brain</a> and our 10 GiB of vectors that use PGVector',
```

**Review by stepashka — APPROVED** [review]
approved the changes, but as i was reviewing left some comments for the adjacent language!

File: content/docs/import/import-sample-data.md (hunk 8 above)
Comment (michrome) [reply content/docs/import/import-sample-data.md:21]: Good point - it's 3 GiB (no s) elsewhere

File: content/docs/import/import-from-neon.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/import/import-from-neon.md:15]: This is true. No critical here.

File: content/docs/import/import-from-postgres.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md:32]: True.

File: content/docs/import/import-from-postgres.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md:99]: True. Good catches.

File: content/docs/import/import-from-postgres.md (hunk 4 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md:115]: Agree.

File: content/docs/import/import-from-neon.md (hunk 1 above)
Comment (stepashka) [reply content/docs/import/import-from-neon.md:15]: @danieltprice what is our preference? GiB or GB when referring to sizes outside of Neon?

File: content/docs/import/import-from-postgres.md (hunk 5 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md:152]: @stepashka I added the database size recommendations with respect to when you use the -j option and psql vs pg_dump. I found this recommended elsewhere. You wouldn't need a parallel load for a small dataset, and avoiding piping for large datasets seems like good general advice. Let me know if you have any concerns.

File: content/release-notes/2022-12-28-console.md (hunk 10 above)
Comment (stepashka) [inline content/release-notes/2022-12-28-console.md:3]:
```suggestion
- API: Added a `branch_logical_size_limit` attribute to the Neon API [Project](https://api-docs.neon.tech/reference/getproject) response body, which shows the logical data size limit for project branches. This attribute replaces the `logical_size_limit` attribute, which was shown in the [Branch](https://api-docs.neon.tech/reference/getprojectbranch) response body. The [Neon Free Tier](/docs/introduction/free-tier/) limits the logical data size of a branch to 3072MiB (3 GiB).
```

Comment (stepashka) [inline content/release-notes/2022-12-28-console.md:3]:
```suggestion
- API: Added a `branch_logical_size_limit` attribute to the Neon API [Project](https://api-docs.neon.tech/reference/getproject) response body, which shows the logical data size limit for project branches. This attribute replaces the `logical_size_limit` attribute, which was shown in the [Branch](https://api-docs.neon.tech/reference/getprojectbranch) response body. The [Neon Free Tier](/docs/introduction/free-tier/) limits the logical data size of a branch to 3072 MiB (3 GiB).
```

File: content/docs/unused/concepts-deprecated [hunk 12]
```diff
@@ -114,7 +114,7 @@ Neon cloud service is available for free during the [Limited and the Technical P
 
 Technical Preview Free Tier users can only create three Projects in Neon. Projects created under Technical Preview Free Tier are subject to additional limits:
 
-- Data size in the Project is limited to 10GB;
+- Data size in the Project is limited to 10GiB;
```
Comment (stepashka) [inline content/docs/unused/concepts-deprecated:117]:
```suggestion
- Data size in the Project is limited to 10 GiB;
```

File: content/docs/import/import-sample-data.md (hunk 8 above)
Comment (stepashka) [inline content/docs/import/import-sample-data.md:21]:
```suggestion
Sample datasets are listed in order of the smallest to largest installed size. Please be aware that the Neon Free Tier has a storage limit of 3 GiBs per branch. Datasets larger than 3 GiB cannot be loaded on the Free Tier.
```

File: content/docs/import/import-sample-data.md (hunk 7 above)
Comment (stepashka) [inline content/docs/import/import-sample-data.md:16]:
```suggestion
- A Neon [Pro](/docs/introduction/pro-plan) account if you intend to install a dataset larger than 3 GiB.
```

**Review by stepashka — COMMENTED** [review]
added all my previous comments as suggestions for convenience

File: content/docs/import/import-from-neon.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/import/import-from-neon.md:15]: OK. Having given this some more thought, I would recommend GB unless the context requires precision, like size calculation instructions. Not everyone is familiar with GiB. Even in those instances, we might use GB, and add a note about GB being "binary gigabytes".

File: content/docs/import/import-from-postgres.md (hunk 2 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md:32]: Removed GiB

File: content/docs/import/import-from-postgres.md (hunk 5 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md:152]: I've removed the specific size classifications.

File: content/docs/import/import-sample-data.md (hunk 7 above)
Comment (danieltprice) [reply content/docs/import/import-sample-data.md:16]: remove the "s"

File: content/release-notes/2022-12-06-console.md (hunk 9 above)
Comment (danieltprice) [reply content/release-notes/2022-12-06-console.md]: Fixed,

File: src/components/pages/ai/testimonials/testimonials.jsx (hunk 11 above)
Comment (danieltprice) [reply src/components/pages/ai/testimonials/testimonials.jsx:16]: I don't think this appears on the site anymore, but fixed.

File: content/release-notes/2022-12-28-console.md (hunk 10 above)
Comment (danieltprice) [reply content/release-notes/2022-12-28-console.md]: fixed

File: content/docs/import/import-sample-data.md (hunk 8 above)
Comment (danieltprice) [reply content/docs/import/import-sample-data.md]: fixed

File: content/docs/import/import-sample-data.md (hunk 7 above)
Comment (danieltprice) [reply content/docs/import/import-sample-data.md]: fixed

File: content/docs/unused/concepts-deprecated (hunk 12 above)
Comment (danieltprice) [reply content/docs/unused/concepts-deprecated:117]: unused.

File: content/docs/import/import-from-postgres.md (hunk 6 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md]: removed specific sizes

File: content/docs/import/import-from-postgres.md (hunk 3 above)
Comment (danieltprice) [reply content/docs/import/import-from-postgres.md]: remove specific sizes

**Review by danieltprice — APPROVED** [review]

**Pull request merged**